3-JSS/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "2-webpack",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "webpack",
+    "start": "webpack serve"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "css-loader": "^6.6.0",
+    "html-webpack-plugin": "^5.5.0",
+    "style-loader": "^3.3.1",
+    "webpack": "^5.68.0",
+    "webpack-cli": "^4.9.2",
+    "webpack-dev-server": "^4.7.4"
+  },
+  "dependencies": {
+    "color": "^4.2.1",
+    "copy-webpack-plugin": "^10.2.4",
```

Comment:
it makes sense to define a webpack plugin `dev dependency`. https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file